README.md

```diff
@@ -44,6 +43,17 @@ Quinn was created and is maintained by by Dirkjan Ochtman and Benjamin Saunders.
 - [ ] Session resumption
 - [ ] HTTP over QUIC
 
+## How to start
+
+The example client [currently always verifies][insecure] the server's certificate chain.
+Example certificates are included in the repository for test purposes.
+
+```sh
+$ cd quinn
+$ cargo run --example server -- --cert ../certs/server.chain --key ../certs/server.rsa .
+$ cargo run --example client -- --ca ../certs/ca.der https://localhost:4433
```

Comment:
The client now needs `--features="dangerous_configuration"`

Reply:
Hmm -- so that doesn't feel like an improvement. Should we remove the insecure option from the example client until the server can actually generate a temporary certificate?